.circleci/config.yml

```diff
@@ -75,9 +75,9 @@ jobs:
           key: cypress-cache-{{ checksum "./package-lock.json" }}
           paths:
             - ~/.cache
-      - run:
-          name: Check dependencies
-          command: bash -i -c 'npm ls'
+              #      - run:
```

Comment:
code comment when this will be uncommented (i.e. when we upgrade/remove transitive dependencies on Angular 8)